src/parsers/css/csstree.js

```diff
@@ -0,0 +1,38 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'css-tree/package.json';
+
+const ID = 'csstree';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage || 'https://github.com/csstree/csstree',
+  locationProps: new Set(['loc']),
+
+  loadParser(callback) {
+    require(['css-tree'], callback);
+  },
+
+  parse(csstree, code) {
+    return JSON.parse(JSON.stringify(csstree.parse(code, {
```

Comment:
It does, but CSSTree AST stores children nodes as a bi-directional list (a List instance), not as an array. If it were possible to make preprocessing of values, it would be easier:

```js
preprocessValue(value) {
    return value instanceof csstree.List ? value.toArray() : value;
}
```

Or we can convert `List` instances to array by walker:

```js
parse(csstree, code) {
    var ast = csstree.parse(code, {
        positions: true
    });
    csstree.walkUp(ast, function(node) {
        if (node.children && node.children instanceof csstree.List) {
            node.children = node.children.toArray();
        }
    });
    return ast;
}
```
It's more verbose, but if you prefer that way I can update the PR ;)
Maybe there will be a method to make it easier out of the box. But I'm still thinking about it.